x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/common/notifications/AbstractAuditor.java

```diff
@@ -158,7 +159,7 @@ protected void indexDoc(ToXContent toXContent) {
                     }
                     backlog.add(toXContent);
                 } else {
-                    logger.error("Latest audit template missing but the back log has been written");
+                    logger.error("Latest audit template missing and audit message cannot be added to the backlog");
```

Comment:
This should never happen, due to the assert on line 155, but since I'm changing the file I thought I'd correct the message anyway.